### ISSUE TITLE
Potential to Release (Some) Pretrained Models?

### ISSUE THREAD

**Issue by daniellawson9999** [danijar/dreamerv3#57]
Hi Danijar,

Great research and implementation. Do you have future plans to release pre-trained model weights for some environments? This could aid in research that aims to study the transfer of world models, mechanistically interpreting components of models, for the purpose of creating offline datasets, and other ideas relating to https://reincarnating-rl.github.io/.

Daniel

**Comment by pannous** [comment]
Alternatively, would anyone be allowed to share their trained weights here, as a kind of first foundation model for RL?

**Comment by danijar** [comment]
Yes, feel free to share models! I don't think I can release them.
